Correct typos and minor changes (#1885)
kartoza/sawps: docs/src/user/manual/user-profile/organisation-page.md

### Email invitation

> **Email Invitation Email**: The individual who is invited to the organisation will receive this email.

Reviewer: ```suggestion
**Email Invitation Email**: The individual being invited to the organisation will receive this email.
```
